modules/invibesBidAdapter.js

```diff
@@ -14,6 +13,8 @@ const CONSTANTS = {
   INVIBES_VENDOR_ID: 436
 };
 
+const storage = getStorageManager(CONSTANTS.INVIBES_VENDOR_ID);
+
 export const spec = {
```

Comment:
hey @invibes , you should define gvlid: INVIBES_VENDOR_ID, in spec object (L18), as gdprEnforcement module is expecting it there, check how other adapters defined it.

Reply:
hi @aleksatr,
 so the storage won't need the gvlid anymore as parameter?

Reply:
hey @aleksatr, I've done the changes.
Thanks!

Reply:
I was talking about gdprEnforcement module, not storageManager.  `getStorageManager` still accepts gvlid, I am talking about adding gvlid to `spec` object of your bid adapter.

Reply:
@aleksatr ok. reverted the getStorageManager to use gvlid and now in spec I've added the givlid.
Thanks!

Reply:
@aleksatr  ok. changed back the storageManager to use gvlid and added to spec the gvlid.

Thanks!